language_detector/main.py

```diff
@@ -2,8 +2,52 @@
 
 from .languages import LANGUAGES
 
+def load_languages(language_list=LANGUAGES):
```

Comment:
While I think making another function is also completely fine, I once read that if you do something more than 3 times, you should make it a function instead of hard coding it each time.

From what I see, you only do load_languages once, and you never change anything in it, you always load your language files, nothing changes there either, so you really wouldn't need to make it a function. 

Not a big deal obviously, but I think it's over complicating it a bit.